core/src/main/scala/cats/data/package.scala

```diff
@@ -3,6 +3,7 @@ package cats
 package object data {
   type NonEmptyStream[A] = OneAnd[Stream, A]
   type ValidatedNel[+E, +A] = Validated[NonEmptyList[E], A]
+  type IorNel[+B, +A] = Ior[NonEmptyList[B], NonEmptyList[A]]
```

Comment:
I am not sure about this definition now. Maybe it should be similar to ValidatedNel

```scala
type IorNel[+B, +A] = Ior[NonEmptyList[B], A]
```

I decided to define it with a NonEmptyList on the right also because I saw some utility on the combine case (or use of Ior's append function to be exact). But when comparing the combine for Either and Validated I think I got it wrong. Ior's flatmap does accumulate errors only if the case is Ior.Both, so it seems appropiate to leave it similar to ValidatedNel's type definition. I would love some feedback on this.